jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/interfaces/IsTextStyled.java

```diff
@@ -0,0 +1,18 @@
+package com.epam.jdi.light.vuetify.interfaces;
+
+import com.epam.jdi.light.common.JDIAction;
+import com.epam.jdi.light.elements.interfaces.base.ICoreElement;
+
+public interface IsTextStyled extends ICoreElement {
+
+    /**
+     * Checks if element is text or not.
+     *
+     * @return {@code true} if element is text, otherwise {@code false}
+     */
+
+    @JDIAction("Check that {name} is text")
+    default boolean isTextStyled() {
```

Comment:
There are examples of "text" description from docs:
- Applies the defined color to text and a low opacity background of the same.
- Makes the background transparent. When using the color prop, the color will be applied to the button text instead of the background.
- Applies the defined color to text and a low opacity background of the same.

It's about some text stylization.

Reply:
Maybe, but docs and JDIAction description are saying that it checks that element has text. And only the method name is about stylization. I think we should have matching method name and descriptions. For example, JDIAction("Check that {name} has styled text"), plus docs providing the description with same meaning.
Or a different method name

Reply:
Fixed naming

About getText() - we have separate TextAssert()